File: westcoast/src/main/java/frc/commands/Pneumatics/ActuateSingleSolenoid.java
package frc.commands.Pneumatics;

import edu.wpi.first.wpilibj.Solenoid;
import edu.wpi.first.wpilibj2.command.CommandBase;

public class ActuateSingleSolenoid extends CommandBase {
  private Solenoid solenoid;

  public ActuateSingleSolenoid(Solenoid solenoid) {
    this.solenoid = solenoid;
  }

  @Override
  public void initialize() {
  }

  @Override
  public void execute() {
    solenoid.set(true);
  }

  @Override
  public void end(boolean interrupted) {
    solenoid.set(false);
  }

  @Override
  public boolean isFinished() {
    return false;
  }
}

File: westcoast/src/main/java/frc/commands/pneumatic/ActuateSingleSolenoid.kt
package frc.commands.pneumatic

import edu.wpi.first.wpilibj.Solenoid
import edu.wpi.first.wpilibj2.command.CommandBase

class ActuateSingleSolenoid(private val solenoid: Solenoid) : CommandBase() {
  override fun initialize() {}
  override fun execute() {
    solenoid.set(true)
  }

  override fun end(interrupted: Boolean) {
    solenoid.set(false)
  }

  override fun isFinished(): Boolean {
    return false
  }
}